core/src/setup/Kokkos_Setup_SYCL.hpp

```diff
@@ -38,6 +33,15 @@
 #include <CL/sycl.hpp>
 #endif
 
+#ifndef KOKKOS_ENABLE_IMPL_SYCL_OUT_OF_ORDER_QUEUES
+#define KOKKOS_IMPL_SYCL_USE_IN_ORDER_QUEUES
+#endif
+
+#if defined(SYCL_EXT_ONEAPI_GRAPH) && \
+    defined(KOKKOS_IMPL_SYCL_USE_IN_ORDER_QUEUES)
+#define KOKKOS_IMPL_GRAPH_SUPPORT
```

Comment:
```suggestion
#define KOKKOS_IMPL_SYCL_GRAPH_SUPPORT
```
Don't you use `KOKKOS_IMPL_SYCL_GRAPH_SUPPORT` everywhere else?